packages/@react-spectrum/dialog/stories/Dialog.stories.tsx

```diff
@@ -49,34 +49,10 @@ storiesOf('Dialog', module)
     'with hero, isDimissable',
     () => renderHero({isDismissable: true, onDismiss: action('dismissed')})
   )
-  .add(
-    'small',
-    () => render({size: 'S'})
-  )
-  .add(
-    'medium',
-    () => render({size: 'M'})
-  )
-  .add(
-    'large',
-    () => render({size: 'L'})
-  )
-  .add(
-    'fullscreen',
-    () => render({size: 'fullscreen'})
-  )
-  .add(
-    'fullscreenTakeover',
-    () => render({size: 'fullscreenTakeover'})
-  )
   .add(
     'form',
     () => renderWithForm({})
   )
-  .add(
```

Comment:
this needs to be added somewhere, it's a good story to demonstrate tabbing order